ReactWindows/ReactNative.Shared.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -103,7 +113,11 @@ public void NetworkingModule_Request_Content_String()
             var waitHandle = new AutoResetEvent(false);
             var module = CreateNetworkingModule(new MockHttpClient(request =>
                 {
+#if WINDOWS_UWP
                     var body = request.Content.ReadAsStringAsync().GetResults();
+#else
```

Comment:
Actually, I will.

---
In reply to: [91630290](https://github.com/ReactWindows/react-native-windows/pull/899#discussion_r91630290) [](ancestors = 91630290,91598112)